src/components/navBar/navBar.js

```diff
@@ -0,0 +1,50 @@
+import data from '../../data/data.json'
+import template from './navBar.pug'
+
+export class NavBar {
+  constructor (node) {
+    this.node = document.querySelector(node)
+    this.elements = {}
+
+    this.fillHtml()
+    this.setElements()
+    this.setNavAnimation()
+    this.setDropDown()
+  }
+
+  setElements () {
+    this.elements.menu = this.node.children[0]
+    this.elements.navbar = this.node.children[2]
+    this.elements.dropDownElmts = this.node.querySelectorAll('.js-dropdown')
+    this.elements.activeElmt = undefined
+  }
+
+  fillHtml () {
+    this.node.innerHTML = template(data)
+  }
+  setNavAnimation () {
+    this.elements.menu.addEventListener('click', () => {
+      this.toggleElmt(this.elements.menu, 'active')
+      this.toggleElmt(this.elements.navbar, 'navbar__list--active')
+      this.toggleElmt(document.body, 'navbar__background')
+    })
+  }
+  toggleElmt (element, data) {
```

Comment:
It's better to use `toggleElement ` instead of `toggleElmt` since it helps to the readability.